src/provider/model/Toast.ts

```diff
@@ -265,12 +265,18 @@ export class Toast implements LayoutItem {
     public async setTransform(transform: ReadonlyRectangle): Promise<void> {
         Object.assign(this._origin, transform.origin);
 
-        return this.trackAnimation((await this._webWindow).setBounds({
+        // TODO [SERVICE-739] This is a fix for setBounds not working on hidden windows.
+        await (await this._webWindow).showAt(transform.origin.x, transform.origin.y);
+
+        await this.trackAnimation((await this._webWindow).setBounds({
             left: transform.origin.x,
             top: transform.origin.y,
             width: transform.size.x,
             height: transform.size.y
         }));
+
+        // TODO [SERVICE-739] This is a fix for setBounds not working on hidden windows.
+        return (await this._webWindow).hide();
```

Comment:
Does this give any kind of flicker? Showing the toast and then hiding again?

I think we show the toast immediately after this anyway. What's the opacity of the toast at this point - does it matter whether the toast is showing here?..

Reply:
This doesn't flicker, the toast's opacity is 0, I could leave it hidden? Main reason I hide it again was for windows 7 (alt+tab).